foo_acfu/embedded/foo_jscript_panel.cpp

```diff
@@ -13,7 +13,7 @@ class Jscript: public Component, public acfu::github_conf {
   Jscript(): Component(guid_jscript, "foo_jscript_panel") {}
 
   static const char* get_owner() {
-    return "marc2k3";
+    return "marc2k3 & kbuffington";
```

Comment:
This is not about credits. It is repository owner name used to construct GitHub API calls to this repository.

Reply:
Fixed